sw_utils/ipfs.py

```diff
@@ -152,3 +154,15 @@ async def fetch(self, ipfs_hash: str) -> bytes:
                 logger.error(e)
 
         raise RuntimeError(f'Failed to fetch IPFS data at {ipfs_hash}')
+
+
+def _strip_ipfs_prefix(ipfs_hash: str) -> str:
+    return ipfs_hash.replace('ipfs://', '').replace('/ipfs/', '')
+
+
+def _to_json(data: bytes) -> list:
```

Comment:
confusing naming, what if data is a correct json dict?

Reply:
That method must be used only for `bytes` type data. If JSON is uploaded, `upload_json` must be used.